lib/components/doc-viewer.jsx

```diff
@@ -66,7 +68,21 @@ export const variants = (metadata, context, route) => {
   return combinations
 }
 
+const slugifyHeaders = (sectionId) => {
```

Comment:
This can be achieved by manipulating the html tree data before it is rendered as described here https://github.com/balena-io/landr/issues/213#issuecomment-513802996